homework_board/apps/frontend/lib/frontend_web/controllers/new_assignment_controller.ex

```diff
@@ -0,0 +1,23 @@
+defmodule FrontendWeb.NewAssignmentController do
+  use FrontendWeb, :controller
+
+  def index(conn, _params) do
+    changeset = Db.Assignments.changeset(%Db.Assignments{})
+    conn
+    |> assign(:changeset, changeset)
+    |> render("index.html")
+  end
+
+  def create(conn, %{"assignments" => %{"title" => title, "description" => description, "priority" => priority_str}} = _params) do
+    {priority, _} = Integer.parse(priority_str)
+
```

Comment:
Could you set the fields when you create the struct:

~~~ elixir
new_assignment = params |> Enum.into(%Db.Assignments{})
~~~